crowdin.yml

```diff
@@ -0,0 +1,3 @@
+files:
+  - source: /src/assets/locales/en/translations.json
+    translation: /src/assets/locales/%two_letters_code%/%original_file_name%
```

Comment:
What's that. Do we use crowdin?

Reply:
apparently I created that file. Together with @hofmann-felix we integrated that translation manager to have a single source of truth of our texts. We pivoted to contentful mostly so I think we don't need an external system for the few ui translations remaining. We can keep that file for now but it's due for deletion. Maybe Felix can provide some more context if it's still useful or not.

Reply:
Since most of the text is in Contentful, I would remove Crowdin as well. I don't think we will change the remaining texts too often in the future.